lib/api/dsl/storage/index.js

```diff
@@ -322,7 +337,7 @@ function addSubfilter(hashKey, filtersObj, subFiltersObj, subfilter) {
   addIndexCollectionToObject(subFiltersObj, filtersObj.index, filtersObj.collection);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 85 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)